### ISSUE TITLE
Module not found

### ISSUE THREAD

**Issue by followbl** [mbrevda/react-image#273]
Attempting to use the new `useImage` hook and cannot import

```
Module not found: Can't resolve 'react-image/useImage'
```

Also either the typings associated with `useImage` or the example in the README needs to be updated...TS is complaining imgPromise is required

```ts
import useImage from 'react-image/useImage'

export default function MyComponent() {
  const {src, isLoading, error} = useImage({
    srcList: 'https://www.example.com/foo.jpg',
  })

  return <img src={src} />
}
```

**Comment by mbrevda** [comment]
Thanks for reporting! I see modules aren't being exported as I had expected them to be.

Trying to figure out how to change this now, probably going to have to release another breaking change and remove the default export...

**Comment by followbl** [comment]
@mbrevda lmk if you'd like me to help testing a change...happy to assist

**Comment by mbrevda** [comment]
That would be great! Try installing `react-image@canary` which is a build of `master` (I didn't get a chance to test that yet so apologies in advance if there as an issue with it)

**Comment by followbl** [comment]
@mbrevda still no luck, pulled down `react-image@canary` and couldn't import the named export `useImage`

**Comment by mbrevda** [comment]
Which version, exactly? The one tagged in `npm view react-image dist-tags.canary` seems to work: https://codesandbox.io/s/shy-grass-1nkxx

**Comment by followbl** [comment]
yup exactly that one - confirmed with the sandbox too `0.0.0-f041bc82bdb574978decadac3cce8cf85539edd4`

<img width="990" alt="Screen Shot 2020-05-15 at 2 02 38 AM" src="https://user-images.githubusercontent.com/927065/82017092-5e305f80-9650-11ea-9d25-7a674cec86b4.png">

**Comment by mbrevda** [comment]
This is strange, it seems to work perfectly fine in that sandbox?
<img width="977" alt="Screen Shot 2020-05-15 at 9 09 26 AM" src="https://user-images.githubusercontent.com/293004/82017455-de27eb00-968b-11ea-8b46-0ac5c05be814.png">

**Comment by rpmonteiro** [comment]
the importing is also broken for me (Typescript)

**Comment by mbrevda** [comment]
Thanks, looking in to this. Specifically, trying to understand why the exports (`export { Img, useImage };`) aren't being recognized.

**Comment by mbrevda** [comment]
Can either of you kindly post a full example? I set u a new project to test using webpack but am not seeing the issue...

**Comment by mbrevda** [comment]
I've temporarily rolled back to 2.4.0 until we can get this resolved

**Comment by 0neSe7en** [comment]
I've created [an example on codesandbox](https://codesandbox.io/s/dazzling-zhukovsky-wepnd?file=/src/App.tsx) with cra and typescript


![image](https://user-images.githubusercontent.com/2234539/82295908-ff1d7280-99e2-11ea-98d3-f23bcf44fa43.png)

**Comment by mbrevda** [comment]
I see the comment, the weird thing is that it works just fine! ([fork](https://codesandbox.io/s/stoic-butterfly-huc17) that adds suspense and a proper image).

I'm at a loss here. Any ideas?

**Comment by mbrevda** [comment]
It seems the issue is mostly with the type definitions. It seems they are fixed in the latest canary( 0.0.0-9442ffb...)?

**Comment by 0neSe7en** [comment]
seems like `Module not found: Can't resolve 'react-image/useImage'` this error already fixed in this commit https://github.com/mbrevda/react-image/commit/a23ffa80429d83f79d40e2ab2c4b4189b692d8fc

I've tried on my project, works well

**Comment by mbrevda** [comment]
Thanks @0neSe7en. Are you using typescript?

**Comment by 0neSe7en** [comment]
yes, I'm using typescript + next.js

**Comment by mbrevda** [comment]
Ok, I'll assume this was fixed then. If any of you are still seeing an issue, please let me know. Wait a few hours before publishing, and then push out v4

**Comment by mbrevda** [comment]
Just published v4.0.0 🥳 
Come back and complain some more if that still doesn't work for y'all!
